docs/src/components/AutoCompleteExample/CustomResults.component.tsx

```diff
@@ -1,37 +1,50 @@
 import * as React from 'react';
 import { ListItem, Check, Typography } from '@retailmenot/anchor';
 
-interface Values {
-    [key: string]: any;
+type Value = {
+    active: boolean;
+    id: string;
+    key: string;
+    label: string;
+    onMouseOver: (event?: React.MouseEvent) => void;
+    onSelect: (event?: React.FocusEvent) => void;
+    [key: string]: any; // Value also contains any new fields passed via the dataSource prop
 }
 
 interface CustomResultsProps {
     index: number;
     currentIndex: number;
     label: string;
-    value?: Values;
+    value: Value;
 }
 
 export const CustomResults = ({
     index,
     currentIndex,
-    label,
-    value,
-}: CustomResultsProps) => {
-    const isLink = value ? value.isLink : false;
-
-    return isLink ? (
+    value: {
+        active,
+        id,
+        isLink, // Here's the isLink key from the dataSource. All custom keys will be added to 'value'.
+        key,
+        label,
+        onMouseOver,
+        onSelect,
+    },
+}: CustomResultsProps) => isLink ? (
         <ListItem
-            key={index}
-            active={index === currentIndex}
+            key={key}
+            active={active}
+            onMouseOver={onMouseOver}
             onSelect={() => {
                 window.open('http://www.google.com', '_blank');
             }}
             prefix={<Check />}
         >
-            <Typography pl="2">Link: {label}</Typography>
+            <Typography pl="2" weight={500}>{label}</Typography>
         </ListItem>
     ) : (
-        <ListItem active={index === currentIndex}>{label}</ListItem>
+        <ListItem active={active} key={key} onSelect={onSelect} onMouseOver={onMouseOver}>
+            <Typography weight={800} mr="2">{label}</Typography>
+            (active: {active.toString()}, currentIndex: {currentIndex}, id: {id}, key: {key} index: {index})
```

Comment:
Also I think it's interesting to see the extra info but not really helpful. And the font looks a little too bold.

Reply:
I agree the info isn't super useful, but I wanted to convey (as clearly as I could) that these are all of the properties available. I can definitely tone down the bold, it's a bit much.